nxrm-aws-resiliency/templates/workdir-daemonset.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name:  {{ .Chart.Name }}-{{ .Chart.Version }}-{{ .Release.Name }}-create-nexus-work-dir
+  namespace: {{ .Values.namespaces.nexusNs }}
+spec:
+  selector:
+    matchLabels:
+      job: dircreator
+  template:
+    metadata:
+      labels:
+        job: dircreator
+    spec:
+      hostPID: true
+      restartPolicy: Always
+      initContainers:
+        # Copy file for creating nexus work directory over and execute it on host 
+        - name: create-nexus-work-dir
+          image: ubuntu:latest
```

Comment:
for the docker image, it is not recommended to use `latest` tag

Reply:
https://github.com/sonatype/nxrm3-helm-repository/pull/41/commits/35fb1119fb025a37ef8acd8ed029b7d3fa06d55e